multiqc/modules/verifybamid/verifybamid.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from VerifyBAMID """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+from multiqc import config
+from multiqc.plots import table
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+    """
+     module class, parses stderr logs.
+    """
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='verifyBAMID', anchor='verifybamid',
+        href='https://genome.sph.umich.edu/wiki/VerifyBamID',
+        info="detect sample contamination and/or sample swap")
+        
+        # dictionary to hold all data for each sample
+        self.verifybamid_data = dict()
+
+        # for each file ending in self.SM
+        for f in self.find_log_files('verifybamid/selfsm'):
+            # add data source to multiqc_sources.txt 
+            self.add_data_source(f)
```

Comment:
This should happen _after_ we've successfully parsed the data, in case spurious files which match the search pattern but don't successfully parse make it this far.